### ISSUE TITLE
Bilinear Filtering just used for Dosbox Purei

### ISSUE THREAD

**Issue by TradeDraft** [schellingb/dosbox-pure#328]
I know that you can change it in Video Options but Turn Off to use the other Cores with it off.

So I reckon that making a Option to Change thru Dosbox Pure Core instead of for All Cores.

**Comment by schellingb** [comment]
This is not something the core has control over. The frontend (RetroArch) decides if it applies any filtering or shaders or whatever.

If you want an option in RetroArch to have per-core video options, you need to send the request their way, for example by opening an item on the [RetroArch issue tracker](https://github.com/libretro/RetroArch/issues).

**Comment by dukeblooders** [comment]
Use Override menu (in core options) to save a configuration for a specific core 😉

**Comment by schellingb** [comment]
Oh, haha! Thanks for posting that @dukeblooders! I thought something like that exists but was only able to find per core control remapping. Would be nice if it were possible to override just a single setting but I guess it's either all or nothing.

**Comment by dukeblooders** [comment]
It saves only the settings which are different from the global configuration (it creates a _core_.cfg file in the RA config folder).
So it can be done manually if necessary.

**Comment by TradeDraft** [comment]
@dukeblooders when you start the Core will it use those Settings without Changing Manually?

**Comment by dukeblooders** [comment]
Yes. Personally, I use it to force a different video driver per core.

**Comment by TradeDraft** [comment]
Thanks @dukeblooders this work Fantasticly.

Really Appreciate the Help
